tom/bot.py

```diff
@@ -24,6 +24,8 @@ def __init__(self, config, secrets, directory, interactive):
         self.default_maintainers = config["reviewers"]
         self.trusted = config["trusted"]
         self.jenkins_repos = config["jenkins_repos"]
+        banned_emails = config.get("banned_emails", {})
+        self.banned_emails = [v for _, v in banned_emails.items()]
```

Comment:
`for v in banned_emails.values()` would be nicer

Reply:
Fixed, thanks!